scripts/runAnalysisOverSamples.py

```diff
@@ -172,6 +218,9 @@ def getFileList(procData,DefaultNFilesPerJob):
 FarmDirectory                      = opt.outdir+"/FARM"
 PROXYDIR                           = FarmDirectory+"/inputs/"
 initProxy()
+doCacheInputs                      = False
+if("IIHE" in localTier): doCacheInputs = True
```

Comment:
This line doesn't work at iihe.
I see two possibilities:
 - use dccp instead of cp (this is the fastest and cleanest solution; see below)
 - instead of accessing dcap://maite.iihe.ac.be:/pnfs/... use /pnfs/... directly (see line 136)

Personnaly, I would modify L128 with:
if("IIHE" in localTier): CopyCommand += "dccp " + l + " " + l[l.rfind('/')+1:] + ";\n" 
else: CopyCommand += "cp " + l + " " + l[l.rfind('/')+1:] + ";\n" 


But tell me what you think :)